src/collectors/postgres/test/testpostgres.py

```diff
@@ -14,43 +14,60 @@ class TestPostgresqlCollector(CollectorTestCase):
     def setUp(self, allowed_names=None):
         if not allowed_names:
             allowed_names = []
+
+        default_config = get_collector_config('PostgresqlCollector', {})
+        self.default_collector = PostgresqlCollector(default_config, None)
```

Comment:
Doesn't look like you ever used this.  Can you add tests using it to test the default config behavior?